editor/export/editor_export_platform.cpp

```diff
@@ -1138,9 +1141,17 @@ Error EditorExportPlatform::export_project_files(const Ref<EditorExportPreset> &
 					return err;
 				}
 
-				//also save the .import file
-				Vector<uint8_t> array = FileAccess::get_file_as_bytes(path + ".import");
-				err = p_func(p_udata, path + ".import", array, idx, total, enc_in_filters, enc_ex_filters, key);
+				// erase useless sections
```

Comment:
```suggestion
				// Erase useless sections.
```
While at it, you could fix the style and casing of `// Erase all PAths` just before the chunk of code you're modifying.

Reply:
Fixed. And I also fixed up the nearby comments too.